### ISSUE TITLE
SelectContent is Missing Styling

### ISSUE THREAD

**Issue by Lermatroid** [shadcn/ui#596]
Hello, I am running into an issue where the SelectContent component seems to be missing it's styling. Currently it looks like this:

![image](https://github.com/shadcn/ui/assets/38444224/b69911ab-36fe-4b22-87c5-43b6c3c354b6)

I have updated my tailwind config and `globals.css` to reflect those in the setup guide, and added the select component using the CLI. I am using it within the Form component from the docs.

Any idea what could be going wrong here? Happy to provide any info needed.

**Comment by dan5py** [comment]
Can you provide more information, like the framework you're using and the configuration files?

**Comment by dan5py** [comment]
I made a simple codesandbox where there's only the select component and it works just fine. Check it [here](https://codesandbox.io/p/sandbox/hopeful-joji-v7fhrx?file=%2Fapp%2Fpage.tsx%3A1%2C58)

**Comment by Lermatroid** [comment]
Ok, found out it was two separate issues:

1. Something with the default font theming was causing anything without a explicit font to default to times new roman. Removing the default theming fixed this issue.
2. I was attempting to use the media class to choose whether to use light or dark theme. I did not realize that shadcn ui only supports class based theming. Updated to class base and now the dark mode on the select is working.

Going to close this issue now. Thanks for the help @dan5py!
